### DISCUSSION TITLE
How to make use of a .dbf file

### DISCUSSION THREAD

**Discussion by DormantVision** [domlysz/BlenderGIS#659] [Q&A]
I am new to BlenderGIS and have read through the wiki

I have an ESRI shapefile, which consists of four files, .shx, .shp, .prj, .dbf

I can "Import" the .shp file easily enough, but I don't understand how to then utilize the .dbf file, which contains the height info

Any help would be much appreciated.

scott

**Comment by domlysz** [comment]
When importing a shapefile you can choose the fields (from the dbf file) which contain elevation values
https://github.com/domlysz/BlenderGIS/wiki/Shapefile-import#elevation-and-extrusion
